### PR TITLE
misc cleanup

### PR DESCRIPTION
**Pull request by slang800**
Random stuff I've found while reading through the code.

PS: What styleguide are we using? The 4-space indents are rather odd for JS :P

### CODE REVIEW

**Comment by satazor** [comment]
The project is linted using `eslint` with this configuration: https://github.com/npms-io/npms-analyzer/blob/master/.eslintrc.json

File: cli.js [hunk 1]
```diff
@@ -25,10 +29,8 @@ yargs
     global: true,
 })
 
-.command('observe', 'Starts observing module changes and pushes them into the queue', require('./cli/observe'))
-.command('consume', 'Consumes modules from the queue, analyzing them', require('./cli/consume'))
-.command('scoring', 'Continuously iterate over the analyzed modules, scoring them', require('./cli/scoring'))
-
-.command('tasks', 'Various useful tasks', require('./cli/tasks'))
-
+.command('observe', 'Starts observing module changes and pushes them into the queue', observe)
```
Comment (satazor) [inline cli.js]: I prefer these requires to be inline, it makes it more obvious when reading.

File: lib/analyze/collect/metadata.js [hunk 2]
```diff
@@ -61,14 +61,12 @@ function normalizeLicense(name, license) {
...
-            license = correctedLicense;
+            return correctedLicense;
         } else {
             log.verbose(logPrefix, `License for module ${name} is not a valid SPDX indentifier`, { license });
-            license = null;
+            return null;
         }
     }
-
-    return license;
```
Comment (satazor) [inline lib/analyze/collect/metadata.js]: This line can't be removed. Also the changes above this line actually makes it worse for me.

Comment (slang800) [inline lib/analyze/collect/metadata.js]: Huh? That should be functionally identical since `license` is scoped to that function, right? ... It just avoids needing to assign back to `license` at the end.

Comment (satazor) [inline lib/analyze/collect/metadata.js]: That function must return the normalized license; if you remove that line, good licenses are not returned (it returns `undefined`).

Comment (slang800) [inline lib/analyze/collect/metadata.js]: Ohh, crap - didn't notice the enclosing if.

**Comment by slang800** [comment]
Ah, you should check out [standard](https://www.npmjs.com/package/standard). The name is incorrect / misleading and the stance on semicolons is relatively uncommon. But other than that, the style matches the Node.js core and most modules.

**Comment by satazor** [comment]
I know standard but the lack of semicolons is a dealbreaker for me. The settings I use are pretty similar to airbnb.

File: cli.js (hunk 1 above)
Comment (satazor) [inline cli.js]: Could you revert the changes on this file? I'm ok merging the rest.

Comment (slang800) [inline cli.js]: reverted! (although I do think that having all the requires at the top of the file makes it easier to read)
